third_party/move/move-model/src/ast.rs

```diff
@@ -569,6 +569,16 @@ pub enum RewriteResult {
     Unchanged(Exp),
 }
 
+/// Visitor position
+pub enum VisitorPosition {
+    Pre,
```

Comment:
```suggestion
    Pre,  // before visiting any children
```

Reply:
I think "subexpressions" is better than children, but good idea.